src/gui/map/map_editor.cpp

```diff
@@ -1066,8 +1066,8 @@ void MapEditorController::createActions()
 	boolean_merge_holes_act = newAction("booleanmergeholes", tr("Merge area holes"), this, SLOT(booleanMergeHolesClicked()), "tool-boolean-merge-holes.png", QString{}, "toolbars.html#area_merge_holes"); // TODO:documentation
 	convert_to_curves_act = newAction("converttocurves", tr("Convert to curves"), this, SLOT(convertToCurvesClicked()), "tool-convert-to-curves.png", QString{}, "toolbars.html#convert_to_curves");
 	simplify_path_act = newAction("simplify", tr("Simplify path"), this, SLOT(simplifyPathClicked()), "tool-simplify-path.png", QString{}, "toolbars.html#simplify_path");
-	cutout_physical_act = newToolAction("cutoutphysical", tr("Cutout"), this, SLOT(cutoutPhysicalClicked()), "tool-cutout-physical.png", QString{}, "toolbars.html#cutout_physical");
-	cutaway_physical_act = newToolAction("cutawayphysical", tr("Cut away"), this, SLOT(cutawayPhysicalClicked()), "tool-cutout-physical-inner.png", QString{}, "toolbars.html#cutaway_physical");
+	clip_area_act = newToolAction("cliparea", tr("Clip area (formerly Cutout)"), this, SLOT(clipAreaClicked()), "tool-clip.png", QString{}, "toolbars.html#cutout_physical");
+	erase_area_act = newToolAction("erasearea", tr("Erase area (formerly Cut away)"), this, SLOT(eraseAreaClicked()), "tool-erase.png", QString{}, "toolbars.html#cutaway_physical");
```

Comment:
I think this somewhat inconvenient for translators:
- First, they will have to translate "Clip area (formerly Cutout)".
- Later, they will have to translate "Clip area".

So for translation, we should use a `tr("Clip area")` IMO. Then the question is how to implement the transition in the UI. 
- We might try something like `tr("%1 (formerly: %2").arg(tr("Clip area"), tr("Cutout"))`. This additional string could be reused for future transitions. However, it might need an extra comment to translators.
- Alternatively, we might drop the old names completely, and just document the change.

Reply:
Thanks, Kai, for the remark. I thought about the translators too and considered the suffix removal a low-effort task and worth the hassle. However, on second thought, I think we can omit the parenthesized text altogether. The new name always comes with the well-known tool icon, so few people will be surprised. Let's go with the new names only and save contributors time.

Another aspect is that a graphical application changes and evolves. It's not an industrial software component that would need compatibility hacks. I'm happy for the opportunity to leave my professional niche and exercise new mental patterns. :)

Reply:
> Let's go with the new names only and save contributors time.

Agreed.